source/partials/primary-domain.md

```diff
@@ -1,5 +1,11 @@
 ### Set a Primary Domain via the Dashboard
 
+<Alert type="danger" title="Warning">
+
+With a Primary Domain set at the platform level, all other domains will be pointed to your Primary domain _at the root level_. If you want to redirect secondary domains to specific pages on your site (for example, `olddomain.com` to `newdomain.com/old-landing-page`), do not set a Primary Domain. Instead use [PHP redirects](/redirects/#redirect-with-php).
+
+</Alert>
+
```

Comment:
@tmountjr @rachelwhitton This is the warning that resolved #5493

Reply:
This is good!